connect-inject/handler.go

```diff
@@ -116,11 +112,7 @@ type Handler struct {
 	// Default metrics settings. These will configure where Prometheus scrapes
 	// metrics from, and whether to run a merged metrics endpoint on the consul
 	// sidecar. These can be overridden via pod annotations.
```

Comment:
This might need to be re-written just a little bit.

Reply:
+1